client/quantum_serverless/utils/storage.py

```diff
@@ -0,0 +1,53 @@
+# This code is a Qiskit project.
+#
+# (C) Copyright IBM 2023.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+=====================================================
```

Comment:
oh, I think this might break sphinx docs :) 

Lines should be the same length, I think

Reply:
weird, the docs test didn't run...